src/globals/core/_lists.scss

```diff
@@ -68,4 +68,25 @@
   .govuk-list--number {
     @extend %govuk-list--number;
   }
+
+  %govuk-list--description > dt {
+    @include govuk-font-bold-19
```

Comment:
We get the font size from `govuk-list`, so this can probably just be `govuk-font-bold`? Also, slight pedantry, but missing the semi-colon.